app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -41,8 +43,14 @@ public void attach(long eventId, IAttendeesView attendeesView) {
     }
 
     @Override
-    public void start() {
-        loadAttendees(false);
+    public void start(boolean forceStart) {
+        if (forceStart)
+            progress = 0;
+        if (progress == 0) {
+            loadAttendees(false);
+        } else {
+            attendeesView.showAttendees(attendeeList);
+        }
```

Comment:
This is when already data is  loaded in presenter, hence just showing it instead of reloading it from disk again.